kubernetes/resource_kubernetes_persistent_volume_v1_test.go

```diff
@@ -0,0 +1,2000 @@
+
+resource "aws_ebs_volume" "test" {
+  availability_zone = "%s"
+  size              = 10
+
+  tags = {
+    Name = %[1]q
+  }
+}
+`, name, region, zone)
+}
+
+func testAccKubernetesPersistentVolumeV1Config_azure_managedDisk(name, location string) string {
+	return fmt.Sprintf(`provider "azurerm" {
+  features {}
+}
+resource "azurerm_resource_group" "test" {
+  name     = %[1]q
+  location = %[2]q
+  tags = {
+    environment = "terraform-provider-kubernetes-test"
+  }
+}
+resource "azurerm_managed_disk" "test" {
+  name                 = %[1]q
+  location             = azurerm_resource_group.test.location
+  resource_group_name  = azurerm_resource_group.test.name
+  storage_account_type = "Standard_LRS"
+  create_option        = "Empty"
+  disk_size_gb         = "1"
+}
+`, name, location)
+}
+
+func testAccKubernetesPersistentVolumeV1Config_azure_PersistentVolumeKindOmitted(name, dataDiskURI string) string {
+	return fmt.Sprintf(`resource "kubernetes_persistent_volume_v1" "test" {
+  metadata {
+    name = %[1]q
+  }
+  spec {
+    capacity = {
+      storage = "1Gi"
+    }
+    access_modes = ["ReadWriteOnce"]
+    persistent_volume_source {
+      azure_disk {
+        caching_mode  = "None"
+        data_disk_uri = %[2]q
+        disk_name     = %[1]q
+      }
+    }
+  }
+}`, name, dataDiskURI)
+}
+
+func testAccKubernetesPersistentVolumeV1Config_azure_PersistentVolumeKind(name, dataDiskURI, kind string) string {
+	return fmt.Sprintf(`resource "kubernetes_persistent_volume_v1" "test" {
+  metadata {
+    name = %[1]q
+  }
+  spec {
+    capacity = {
+      storage = "1Gi"
+    }
+    access_modes = ["ReadWriteOnce"]
+    persistent_volume_source {
+      azure_disk {
+        caching_mode  = "None"
+        data_disk_uri = %[2]q
+        disk_name     = %[1]q
+        kind          = %[3]q
+      }
+    }
+  }
+}`, name, dataDiskURI, kind)
+}
+
+func testAccKubernetesPersistentVolumeV1Config_azure_blobStorage(name, location string) string {
+	return fmt.Sprintf(`provider "azurerm" {
+  features {}
+}
+resource "azurerm_resource_group" "test" {
+  name     = "%s"
+  location = "%s"
+  tags = {
+    environment = "terraform-provider-kubernetes-test"
+  }
+}
+resource "azurerm_storage_account" "test" {
+  name                     = %[1]q
+  resource_group_name      = azurerm_resource_group.test.name
+  location                 = azurerm_resource_group.test.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+  account_kind             = "BlobStorage"
+  tags = {
+    environment = "terraform-provider-kubernetes-test"
+  }
+}
+resource "azurerm_storage_container" "test" {
+  name                  = %[1]q
+  storage_account_name  = azurerm_storage_account.test.name
+  container_access_type = "private"
+}
+`, name, location)
+}
+
+func testAccKubernetesPersistentVolumeV1Config_azure_PersistentVolumeAzureFile(name, secretName string) string {
+	return fmt.Sprintf(`resource "kubernetes_persistent_volume" "test" {
```

Comment:
Please change `kubernetes_persistent_volume` to `kubernetes_persistent_volume_v1`.